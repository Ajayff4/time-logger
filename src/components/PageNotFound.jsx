import React from 'react'

export default function PageNotFound() {
    return (
        <div id="errorPage404Component">
            <center>
                <h1 id="text404">Page Not Found 404</h1>
            </center>
        </div>        
    )
}