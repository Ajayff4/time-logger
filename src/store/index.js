import { createStore, applyMiddleware } from 'redux'
import rootReducer from '../reducers/index'
//import logger from 'redux-logger'
import { composeWithDevTools } from 'redux-devtools-extension'
import rootSaga from '../saga'
import createSagaMiddleware from 'redux-saga'

const sagaMiddleware = createSagaMiddleware()
const store = createStore(
    rootReducer,
    composeWithDevTools(
        applyMiddleware(sagaMiddleware))
)

sagaMiddleware.run(rootSaga)

export default store