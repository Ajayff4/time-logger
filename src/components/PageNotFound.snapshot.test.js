import React from 'react'
import renderer from 'react-test-renderer'
import PageNotFound from './PageNotFound'
import { Provider } from 'react-redux';
import store from '../store';
import { BrowserRouter as Router } from 'react-router-dom';

describe('PageNotFound Snapshot Testing...', () => {
    it('renders component correctly', () => {
        const tree = renderer.create(<Provider store={store}><Router><PageNotFound /></Router></Provider> ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});