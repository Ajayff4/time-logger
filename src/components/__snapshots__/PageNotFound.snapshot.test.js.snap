// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound Snapshot Testing... renders component correctly 1`] = `
<div
  id="errorPage404Component"
>
  <center>
    <h1
      id="text404"
    >
      Page Not Found 404
    </h1>
  </center>
</div>
`;
